packages/vue/src/components/molecules/SfPagination/SfPagination.vue

```diff
@@ -54,9 +63,16 @@
         </component>
       </slot>
     </template>
-    <template v-if="showLast">
-      <slot v-if="lastVisiblePageNumber < total - 1" name="points">
-        <div class="sf-pagination__item">...</div>
+    <template>
+      <slot name="points">
+        <div
+          :class="{
+            'display-none': !showLast && lastVisiblePageNumber < total - 1,
+          }"
+          class="sf-pagination__item"
+        >
+          ...
+        </div>
```

Comment:
There is probably sth wrong with this condition  
![Screenshot from 2021-11-16 10-59-39](https://user-images.githubusercontent.com/32042425/141964131-e113b094-a668-487b-9484-5e69088420c2.png)